### ISSUE TITLE
V10.x differences in folder/filenames

### ISSUE THREAD

**Issue by garywong-bc** [bcgov/automated-testing#80]
So according to:
https://docs.cypress.io/guides/references/migration-guide#Migrating-to-Cypress-version-10-0

many of the naming conventions within https://github.com/rstens/cypress-template/tree/dev/cypress will have to change (assuming we want v10.x to be the lowest release we want to promote).

**Comment by garywong-bc** [comment]
```bash
❯ docker run -it -v ${PWD}/cypress:/e2e -w /e2e  -e ELECTRON_EXTRA_LAUNCH_ARGS cypress/included:10.7.0 --spec cypress/integration/examples/utilities.spec.js --browser chrome

There is a cypress.json file at the path: /e2e

Cypress version 10.0.0 no longer supports cypress.json.

Please run cypress open to launch the migration tool to migrate to cypress.config.{js,ts,mjs,cjs}.

https://on.cypress.io/migration-guide
```

**Comment by garywong-bc** [comment]
Agree that the baseline version we should start from is `v10.7.0`, as this is the current version and incidentally this includes Cypress Studio.
